<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Border</class>
 <widget class="QDialog" name="Border">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>187</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>227</width>
    <height>187</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>227</width>
    <height>187</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>136</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(169, 217, 215);
  border: 1px solid gray;
border-radius: 5px;
color:  black;</string>
   </property>
   <property name="text">
    <string>Толщина </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="color_border">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>95</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 1px solid gray;
border-radius: 5px;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="type_border">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>50</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
border: 1px solid gray;
border-radius: 5px;
background: rgb(228, 227, 229);
font-size: 16px;
color: black;
}</string>
   </property>
   <item>
    <property name="text">
     <string>―</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>┅</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>50</y>
     <width>136</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(169, 217, 215);
  border: 1px solid gray;
border-radius: 5px;
color:  black;</string>
   </property>
   <property name="text">
    <string>Тип </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_border">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>140</y>
     <width>217</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
   </property>
   <property name="text">
    <string>Свернуть</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="thickness_border">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>5</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
border: 1px solid gray;
border-radius: 5px;
background: rgb(228, 227, 229);
font-size: 16px;
color: black;
}

</string>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>20.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.250000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_color">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>95</y>
     <width>136</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(169, 217, 215);
border: 1px solid gray;
border-radius: 5px;
color:  black;
</string>
   </property>
   <property name="text">
    <string>Цвет</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
